### PR TITLE
Feature/homework 13 -> Create tests

### PR DESCRIPTION
**Pull request by Hlb03**
Created tests to the Homework 9 (Hibernate).
Per each service method two test cases were created. Third case (corner case) wasn't covered, because there are only basic params in each service method.

### CODE REVIEW

File: src/test/java/com/example/task_2/hibernate_service_test/HibernateUserServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    }
+
+    @Test
+    public void updateUserNameAndCreateTicket() {
+        Ticket ticketToStore = new Ticket(20L);
+
+        doNothing().when(userDAO).updateUserNameAndSaveTicket(1L, "new Name", ticketToStore);
+        userService.updateUserNameAndCreateTicket(1L, "new Name", ticketToStore);
+
+        verify(userDAO, times(1)).updateUserNameAndSaveTicket(1L, "new Name", ticketToStore);
```
Comment (deolexx) [inline src/test/java/com/example/task_2/hibernate_service_test/HibernateUserServiceTest.java:123]: just reminder that if you want verify that method called once you just could write verify(userDao).update...

**Review by deolexx — APPROVED** [review]
